app/controllers/admin/participants_controller.rb

```diff
@@ -10,8 +10,10 @@ class ParticipantsController < Admin::BaseController
     def show; end
 
     def index
-      @participant_profiles = policy_scope(ParticipantProfile)
-        .ransack(user_full_name_or_school_name_or_school_urn_cont: params[:query]).result
```

Comment:
Why was this change needed?

Reply:
Cause `school` is no longer an association on `ParticipantProfile`, so `ransack` does not work :(